ios/Classes/SwiftHmssdkFlutterPlugin.swift

```diff
@@ -786,6 +789,56 @@ public class SwiftHmssdkFlutterPlugin: NSObject, FlutterPlugin, HMSUpdateListene
         }
         )
     }
+    
+    private func setPlaybackAllowedForTrack(_ call: FlutterMethodCall, _ result: @escaping FlutterResult){
+        let arguments = call.arguments as! [AnyHashable: Any]
+
+        guard let isPlaybackAllowed = arguments["is_playback_allowed"] as? Bool,
+              let trackID = arguments["track_id"] as? String,
+              let trackKind = arguments["track_kind"] as? String
+        else {
+            result(HMSErrorExtension.getError("Invalid arguments passed in \(#function)"))
+            return
+        }
+        
+        if(kind(from: trackKind) == HMSTrackKind.audio){
```

Comment:
Same as above here. `getAllAudioTracks` & `getAllVideoTracks` util functions are provided from SDKs. Use that instead of this